lib/index.js

```diff
@@ -239,4 +239,5 @@ const structErrors = r => {
 // Exports
 // -------
 
-module.exports = { version:'1.0.0-rc', Url, RawUrl }
\ No newline at end of file
+export Url
+export RawUrl
```

Comment:
Should this be export `{ Url, RawUrl }` instead? The double export doesn't work for me with node, and esbuild. 
I'm only learning the module syntax now, so I may well be missing something.

Reply:
Whoops, you're right!